v2/en/events.md

```diff
@@ -0,0 +1,15 @@
+<!---
+    @title         Events
+    @creator       Yichun Zhang
+    @modifier      Yichun Zhang
+    @created       2016-11-28 15:58 GMT
+    @changes       1
+--->
+
+
+#  English Events
+
+#  Chinese Events
+* OpenResty Con 2016 (2016-12-10 9:00M at Shenzhen, China)
```

Comment:
I think `10 December 2016` is better than `2016-12-10`.

Also, "9:00M" => "9:00 AM +0800". You missed the `A` letter. And also it's better to specify a time zone.